twisted/python/compat.py

```diff
@@ -39,6 +40,10 @@
 else:
     _PY3 = True
 
+if platform.python_implementation() == 'PyPy':
+    _PYPY = True
+else:
+    _PYPY = False
```

Comment:
It does. I was mimicking the _PY3 check in the same file in an attempt to
keep the style consistent. I'm happy to change it, though.

On Friday, July 8, 2016, Alex Gaynor <notifications@github.com> wrote:

> In twisted/python/compat.py
> <https://github.com/twisted/twisted/pull/221#discussion_r70076196>:
>
> > @@ -39,6 +40,10 @@
> >  else:
> >      _PY3 = True
> >
> > +if platform.python_implementation() == 'PyPy':
> > +    _PYPY = True
> > +else:
> > +    _PYPY = False
>
> This simplifies to _PYPY = platform.python_implementation() == 'PyPy'
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/twisted/twisted/pull/221/files/6dae2825a56520e9011af4844a41800121bc3ea7#r70076196>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAq-9wPytqbiSVxBJwfvap4eTIFd2_Cbks5qTlVngaJpZM4I-feb>
> .
>